src/main/java/com/example/MainTask2.java

```diff
@@ -0,0 +1,35 @@
+//2.Отобразить в окне консоли аргументы командной строки в обратном порядке.
+
+package com.example;
+
+import java.util.Scanner;
+
+public class MainTask2 {
+    public static final String SEPARATOR = " ";
+
+    public static void main(String[] args) {
+        String argumentsString = readString();
+
+        String[] words = splitString(argumentsString);
+
+        printReversed(words);
+    }
+
+    private static String readString() {
+        Scanner scanner = new Scanner(System.in);
+        String argumentsString = scanner.nextLine();
+        scanner.close();
+        return argumentsString;
+    }
+
+    private static String[] splitString(String argumentsString) {
+        return argumentsString.split(SEPARATOR);
+    }
+
+    private static void printReversed(String[] args) {
+        for (int i = args.length; i > 0; i--) {
+            System.out.print(args[i-1] + " ");
```

Comment:
Please replace with spaces inside of square brackets: `System.out.print(args[i - 1] + " ")`
You can use standard IDEA formatter using shortcut ctrl+alt+L